content/articles/account-activation.markdown

```diff
@@ -14,9 +14,8 @@ Activation is easy and includes a free 30-day trial. You can cancel at any time
 <div class="section-steps" markdown="1">
 ##### To activate your account
 
-1. Log in and select <label>choose plan</label>.
-1. Choose either <label>Individual</label> or <label>Business</label>, and click <label>Continue</label>.
 1. Decide which plan has the best features for your needs, and click <label>Begin 30 day trial</label>.
+1. Create your account.
```

Comment:
This could be a little more descriptive, no?

Reply:
👍 Done on [f5cb09c](https://github.com/dnsimple/dnsimple-support/pull/991/commits/f5cb09c5c67ffe937c0fb98016c4d779428cf78a)